Binary files xchat-2.8.4.orig/src/common/.text.c.swp and xchat-2.8.4/src/common/.text.c.swp differ
diff --speed-large-files --minimal -Nru xchat-2.8.4.orig/src/fe-text/fe-text.c xchat-2.8.4/src/fe-text/fe-text.c
--- xchat-2.8.4.orig/src/fe-text/fe-text.c	2007-04-29 02:07:09.000000000 -0400
+++ xchat-2.8.4/src/fe-text/fe-text.c	2007-10-04 01:55:56.719012000 -0400
@@ -106,7 +106,7 @@
 				" \017This binary compiled \00310"__DATE__"\017\n",
 				get_cpu_str(),
 				glib_major_version, glib_minor_version, glib_micro_version);
-	fe_print_text (sess, buf);
+	fe_print_text (sess, buf, 0);
 
 	fe_print_text (sess, "\n\nCompiled in Features\0032:\017 "
 #ifdef USE_PLUGIN
@@ -121,7 +121,7 @@
 #ifdef USE_IPV6
 	"IPv6"
 #endif
-	"\n\n");
+	"\n\n", 0);
 	fflush (stdout);
 	fflush (stdin);
 }
@@ -146,7 +146,7 @@
 static const short colconv[] = { 0, 7, 4, 2, 1, 3, 5, 11, 13, 12, 6, 16, 14, 15, 10, 7 };
 
 void
-fe_print_text (struct session *sess, char *text)
+fe_print_text (struct session *sess, char *text, time_t stamp)
 {
 	int dotime = FALSE;
 	char num[8];
